docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/github/index.md

```diff
@@ -174,6 +181,85 @@ For additional information on configuring webhooks, refer directly to the
 [GitHub Docs](https://docs.github.com/en/webhooks/using-webhooks/creating-webhooks).
 :::
 
+### Webhooks from an Organization
+
+To configure an organization to notify the receiver of relevant events from all
+repositories in the organization:
+
+1. Navigate to `https://github.com/organizations/<organization>/settings/hooks`,
+   where `<organization>` has been replaced with the name of an organization for
+   which you are an administrator.
+
+    ![Settings](./img/repos/settings.png "Settings")
+
+1. Click <Hlt>Add webhook</Hlt>.
+
+1. Complete the <Hlt>Add webhook</Hlt> form:
+
+    ![Add Webhook Form](./img/orgs/add-webhook-form.png "Add Webhook Form")
+
+    1. Complete the <Hlt>Payload URL</Hlt> field using the URL
+       [for the webhook receiver](#retrieving-the-receivers-url).
+
+    1. Set <Hlt>Content type</Hlt> to `application/json`.
+
+    1. Complete the <Hlt>Secret</Hlt> field using to the (unencoded) value
+       assigned to the `secret` key of the `Secret` resource referenced by the
+       [webhook receiver's configuration](#configuring-the-receiver).
+
+    1. Under <Hlt>Which events would you like to trigger this webhook?</Hlt>:
+
+        Leave <Hlt>Just the push event.</Hlt> selected, unless you would
+        like to receive events when container images or Helm charts are
+        pushed to associated GHCR repositories.
+
+        To receive such events, select
+        <Hlt>Let me select individual events.</Hlt>, then ensure
+        <Hlt>Pushes</Hlt> and <Hlt>Packages</Hlt> are both selected.
+
+        :::note
+        You will only receive events from those GHCR repositories explicitly
+        associated with your Git repository.
+
+        For more information on this topic, refer to
+        [these GitHub docs](https://docs.github.com/en/packages/learn-github-packages/connecting-a-repository-to-a-package).
+        :::
+
+    1. Ensure <Hlt>Active</Hlt> is selected.
+
+    1. Click <Hlt>Add webhook</Hlt>.
+
+1. Verify connectivity:
+
+    1. From the organization <Hlt>Webhooks</Hlt> dashboard, select the new webhook.
+
+        ![Webhooks](./img/orgs/webhooks.png "Webhooks")
+
+    1. Scroll down to the <Hlt>Recent Deliveries</Hlt> section.
+
+        ![Recent Deliveries](./img/orgs/recent-deliveries.png "Recent Deliveries")
+
+    1. Select the <Hlt>ping</Hlt> event and ensure an HTTP response status of
+       `200` was received from the webhook receiver.
+
+        ![Ping Response](./img/orgs/ping-response.png "Ping Response")
+
+When these steps are complete, all repositories in the organization will send
+events to the webhook receiver.
+
+:::warning
+Organization-level webhooks will receive events from **all** repositories in
+the organization. This includes both public and private repositories. Ensure
+that your webhook receiver is configured to handle the volume of events and
+that you have appropriate security measures in place.
+:::
+
```

Comment:
This was mentioned in the introductory text. I don't believe it needs to be said again.

```suggestion
```